README.md

```diff
@@ -27,29 +28,30 @@ Once the required ones are set, you can just run the following to log in:
 
 You can also export variables and run it all in one line:
 
-```NPM_USER=testUser NPM_PASS=testPass NPM_EMAIL=test@example.com npm-cli-login```
+    NPM_USER=testUser NPM_PASS=testPass NPM_EMAIL=test@example.com npm-cli-login
 
 There is also support for command line arguments:
 
-- ```-u```: NPM Username
-- ```-p```: NPM Password
-- ```-e```: NPM Email
-- ```-r```: NPM Registry
-- ```-s```: NPM Scope
-- ```--quotes```: Set to ```false``` by default. Specifies wheather your auth token requires quotes. This might required when your auth token has special characters, like ```=```, ```?``` etc.
+- `-u`: NPM Username
+- `-p`: NPM Password
+- `-e`: NPM Email
+- `-r`: NPM Registry
+- `-s`: NPM Scope
+- `--quotes`: Set to `false` by default. Specifies whether your auth token requires quotes. This might required when your auth token has special characters, like `=`, `?` etc.
+- `--config-path`: Set to `~/` by default. Can be used to configure a custom .npmrc file to edit
 
-For example: ```npm-cli-login -u testUser -p testPass -e test@example.com```
+For example: `npm-cli-login -u testUser -p testPass -e test@example.com`
```

Comment:
Please use the triple backticks, so that the command does not look cramped.

Or take it to a new line and add four spaces, so that it's indented